recipe/conda_build_config.yaml

```
@@ -0,0 +1,5 @@
+channel_targets:
+  - conda-forge compat-test
```

In the spirit of [CFEP-05](https://github.com/conda-forge/cfep/blob/main/cfep-05.md), let's prefix this with the feedstock name:

```suggestion
  - conda-forge zlib-ng-compat-test
```